SpatialGDK/Documentation/content/cloud-deployment-workflow.md

```diff
@@ -71,13 +71,16 @@ You can launch a cloud deployment using the Unreal Editor or the SpatialOS CLI.
 4. In the **Deployment Name** field, enter a name for your deployment. This labels the deployment in the [Console]({{urlRoot}}/content/glossary#console).
 5. Leave the Snapshot File field as it is. In the **Launch Config File** field, enter the path to the launch configuration file for this deployment (including the file name).
 6. (Optional) If needed, change the **Region**.
-7. (Optional) Create an additional deployment with [simulated players]({{urlRoot}}/content/simulated-players) that connect to your main game deployment. Simulated players are game clients running in the cloud, mimicking real players of your game from a connection flow and server-worker load perspective. This means they’re useful for scale testing. 
-
-    To create an additional deployment with simulated players, in the **Simulated Players** section:
+7. (Optional) Create an additional deployment with [simulated players]({{urlRoot}}/content/simulated-players) that connect to your main game deployment. Simulated players are game clients running in the cloud, mimicking real players of your game from a connection flow and server-worker load perspective. This means they’re useful for scale testing. </br></br>
+    Build out the simulated player clients (which will run on Linux in the cloud) using the following command:</br></br>
+    ```
+    BuildWorker.bat <YourProject>SimulatedPlayer Linux Development <YourProject>.uproject 
```

Comment:
We should probably include both the engine plugin and project plugin alternatives, similar to what we do above for server and client worker assemblies.

Reply:
For each of the 3 docs pages in this PR, we explain the full filepath of Buildworker.bat above (when we explain how to build the client and server worker assemblies) 

By this point people will know where their buildworker.bat file is so I feel we don't need to re-explain it

Reply:
Actually you're right for consistency let's keep listing the engine and project plugin filepaths - updated